### PR TITLE
 app-admin/dio: fix build with sys-libs/ncurses[tinfo]

### PR DESCRIPTION
**Pull request by ConiKost**
app-admin/dio: fix build with sys-libs/ncurses[tinfo]

Also bumped to EAPI=6

Closes: https://bugs.gentoo.org/646802
Package-Manager: Portage-2.3.43, Repoman-2.3.10

### CODE REVIEW

File: app-admin/dio/dio-1.5.2-r1.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
```
Comment (Zlogene) [inline app-admin/dio/dio-1.5.2-r1.ebuild]: why 6?

File: app-admin/dio/dio-1.5.2-r1.ebuild [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+S="${WORKDIR}/${P}/src"
+
+DOCS=( "README" "../README.md" )
+HTML_DOCS=( "../site/dio.png" "../site/www.diodio.org" )
+
+src_prepare() {
+	# Include the 'tinfo' lib, if sys-libs/ncurses is compiled with USE="tinfo"
+	if has_version sys-libs/ncurses[tinfo]; then
+		sed -e 's/lcurses/& -ltinfo/' -i Makefile
```
Comment (Zlogene) [inline app-admin/dio/dio-1.5.2-r1.ebuild]: `|| die`

File: app-admin/dio/dio-1.5.2-r1.ebuild (hunk 1 above)
Comment (ConiKost) [reply app-admin/dio/dio-1.5.2-r1.ebuild]: Can I go to 7 directly? I read somewhere, that not all classes are ready for 7?

File: app-admin/dio/dio-1.5.2-r1.ebuild (hunk 2 above)
Comment (ConiKost) [reply app-admin/dio/dio-1.5.2-r1.ebuild]: Fixed.

File: app-admin/dio/dio-1.5.2-r1.ebuild (hunk 1 above)
Comment (ConiKost) [reply app-admin/dio/dio-1.5.2-r1.ebuild]: Anyway, changed to `EAPI=7`, as it seems to work.

File: app-admin/dio/dio-1.5.2-r1.ebuild [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="sys-libs/ncurses:0="
+DEPEND="${RDEPEND}"
+
+S="${WORKDIR}/${P}/src"
+
+DOCS=( "README" "../README.md" )
+HTML_DOCS=( "../site/dio.png" "../site/www.diodio.org" )
```
Comment (mgorny) [inline app-admin/dio/dio-1.5.2-r1.ebuild]: What's the purpose of `dio.png`?

Comment (ConiKost) [reply app-admin/dio/dio-1.5.2-r1.ebuild]: I thought, it belongs to the html docs, but apparently not, so dropped.

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**